trajectory_tracker/test/src/test_trajectory_tracker.cpp

```diff
@@ -33,6 +33,8 @@
 
 #include <trajectory_tracker_test.h>
 
+#include <angles/angles.h>
```

Comment:
It's installed by an indirect dependency.
Please add `test_depend` to angles, replace it by `tf2::Quaternion::angleShortestPath()` or implement `angleShortestPath` here.

Reply:
`tf2::Quaternion::angleShortestPath() is used instead of angles.`